ecosystem/python/sdk/examples/fee_payer_transfer_coin.py

```diff
@@ -0,0 +1,108 @@
+# Copyright © Aptos Foundation
+# SPDX-License-Identifier: Apache-2.0
+
+import asyncio
+
+from aptos_sdk.account import Account
+from aptos_sdk.async_client import ApiError, FaucetClient, ResourceNotFound, RestClient
+from aptos_sdk.authenticator import Authenticator, FeePayerAuthenticator
+from aptos_sdk.bcs import Serializer
+from aptos_sdk.transactions import (
+    EntryFunction,
+    FeePayerRawTransaction,
+    SignedTransaction,
+    TransactionArgument,
+    TransactionPayload,
+)
+
+from .common import FAUCET_URL, NODE_URL
+
+
+async def main():
+    # :!:>section_1
+    rest_client = RestClient(NODE_URL)
+    faucet_client = FaucetClient(FAUCET_URL, rest_client)  # <:!:section_1
+
+    # :!:>section_2
+    alice = Account.generate()
+    bob = Account.generate()
+    sponsor = Account.generate()  # <:!:section_2
+
+    print("\n=== Addresses ===")
+    print(f"Alice: {alice.address()}")
+    print(f"Bob: {bob.address()}")
+    print(f"Sponsor: {sponsor.address()}")
+
+    # :!:>section_3
+    await faucet_client.fund_account(sponsor.address(), 100_000_000)  # <:!:section_3
+
+    print("\n=== Initial Data ===")
+    # :!:>section_4
+    try:
+        alice_sequence_number = await rest_client.account_sequence_number(
+            alice.address()
+        )
+        raise Exception("This should never be accessed")
+    except ApiError:
+        alice_sequence_number = 0
+
+    try:
+        bob_balance = await rest_client.account_balance(bob.address())
+        raise Exception("This should never be accessed")
+    except ResourceNotFound:
+        bob_balance = None
```

Comment:
FYI, this is a perfect example of how `account_balance` might want to instead just return 0 or None rather than having to wrap it like this

Reply:
hmm... I had been pondering that, but it gets into the nuances of handling different exceptions, which I didn't want to commit to in the SDK just yet.